packages/math/texlike.lua

```diff
@@ -401,6 +401,9 @@ local function convertTexlike (_, content)
   return ret
 end
 
+registerCommand("%", {}, function ()
```

Comment:
I haven't seen the changes in the texlike parser that allow to parse `\%`. Am I missing something?

Reply:
`\%` was already parsed as a command. If I read the grammar correctly, this is because the `ctrl_sequence_name` rule has two cases, one of them (`ctrl_word`) accepts alphabetic command names of any length, the other (`ctrl_symbol`) accepts any one-character-long command name except from `{}\`, therefore including `\%`.

Reply:
OK, I wasn't sure.